src/menu/menu.service.ts

```diff
@@ -12,7 +12,8 @@ export class MenuService {
     private prisma: PrismaService,
   ) {}
   async createMenuItem(dto: MenuDto, userInfo: UserInfo) {
```

Comment:
let's rename UserInfo to AuthUser - meaning that it is user in JWT token.

Please do not use Info appendix, if it is already clear what the object is about. 

Here, we added context Auth as prefix. this is much better, since we not only know that it has user info, we now also know that it is user info related to auth